guides/common/modules/proc_configuring-dhcpd-to-use-with-server.adoc

```diff
@@ -0,0 +1,205 @@
+[id="configuring-dhcpd-to-use-with-server"]
+= Configuring dhcpd to use with {ProductName}
+
+To configure an external DHCP server running {EL} to use with {ProductName}, you must install the ISC DHCP Service and Berkeley Internet Name Domain (BIND) utilities packages.
+You must also share the DHCP configuration and lease files with {ProductName}.
+The example in this procedure uses the distributed Network File System (NFS) protocol to share the DHCP configuration and lease files.
+
+ifdef::foreman-deb[]
+[NOTE]
+====
+This procedure describes how to run a remote ISC DHCP server on {EL} 9.
+====
+endif::[]
+
+
+.Procedure
+. Perform the following steps on the DHCP server:
+
+.. Install the required packages:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {client-package-install-el8} dhcp-server bind-utils
+----
+
+.. Generate a security token:
++
+[options="nowrap" subs="+quotes"]
+----
+# tsig-keygen -a hmac-md5 _omapi_key_
+key "omapi_key" {
+	algorithm hmac-md5;
+	secret "4z1jwYO0RGUTJbWDepFBdg==";
+};
+----
+
+.. Edit the `/etc/dhcp/dhcpd.conf` file for all subnets, and add the key generated by `tsig-keygen`.
+The following is an example:
++
+[options="nowrap" subs="+quotes"]
+----
+# cat /etc/dhcp/dhcpd.conf
+default-lease-time 604800;
+max-lease-time 2592000;
+log-facility local7;
+
+subnet _192.168.38.0_ netmask _255.255.255.0_ {
+	range _192.168.38.10 192.168.38.100_;
+	option routers _192.168.38.1_;
+	option subnet-mask _255.255.255.0_;
+	option domain-search "_virtual.lan_";
+	option domain-name "_virtual.lan_";
+	option domain-name-servers _8.8.8.8_;
+}
+
+omapi-port 7911;
+key _omapi_key_ {
+	algorithm hmac-md5;
+	secret "_<key_secret>_";
+};
+omapi-key _omapi_key_;
+----
++
+Note that the `option routers` value is the IP address of your {ProjectServer} or {SmartProxyServer} that you want to use with an external DHCP service.
+
+.. Open the DHCP port in the `firewalld` service:
++
+[options="nowrap"]
+----
+# firewall-cmd --add-service dhcp
+----
+
+.. Make the changes persistent:
++
+[options="nowrap"]
+----
+# firewall-cmd --runtime-to-permanent
+----
+
+. Perform the following steps on {ProjectServer}:
+
+.. Determine both the UID and the primary GID of the `foreman-proxy` user:
```

Comment:
There is only a single step mentioned, so can we merge these? Then answering my own question: as I suspected in https://github.com/theforeman/foreman-documentation/pull/3530#discussion_r2060134958, it was always pointless.

We relied on world readable files because it calls `chmod o+rx /etc/dhcp/` and `chmod o+r /etc/dhcp/dhcpd.conf` instead of `chown` or `chgrp` or ACL equivalents. Creating the user and group was always pointless.

I'm not sure if I should point out here that `dhcpd.conf` contains the credentials to make changes via omapi.

My suggestion is now to simply drop the user instructions (since they're not needed) and I'll further pick this up. Let's not waste too much time on this whole chapter.